qiita_db/user.py

```diff
@@ -509,6 +525,68 @@ def _change_pass(self, newpass):
             TRN.add(sql, [hash_password(newpass), self._id])
             TRN.execute()
 
+    def messages(self, count=None):
+        """Return messages in user's queue
+
+        Parameters
+        ----------
+        count : int, optional
+            Number of messages to return, starting with newest. Default all
+
+        Returns
+        -------
+        list of tuples
+            Messages in the queue, in the form
+            [(msg_id, msg, timestamp, read), ...]
+        """
+        with TRN:
+            sql_info = [self._id]
+            sql = """SELECT message_id, message, message_time, read
+                     FROM qiita.message_user
+                     JOIN qiita.message USING (message_id)
+                     WHERE email = %s ORDER BY message_time DESC"""
+            if count is not None:
+                sql += " LIMIT %s"
+                sql_info.append(count)
+            TRN.add(sql, sql_info)
+            return TRN.execute_fetchindex(-1)
```

Comment:
The `-1` is not needed, remove for consistency with other codebase and for magic numbers.